Dockerfile

```diff
@@ -68,10 +68,11 @@ RUN git clone https://github.com/tfutils/tfenv.git ~/.tfenv \
  && tfenv use ${TERRAFORM_VERSION}
 
 # pip installs
-COPY awscli.txt .
+COPY awscli.txt checkov.txt .
 ENV PATH="/root/.local/bin:${PATH}"
 RUN python3 -m pip install --upgrade --no-cache-dir pip \
- && pip install --user --no-cache-dir -r awscli.txt
+ && pip install --user --no-cache-dir -r awscli.txt \
```

Comment:
I would have just done `-r awscli.txt -r checkov.txt` but they depend on conflicting versions of `boto`